README.md

```diff
@@ -1,50 +1,48 @@
-2. [Usage](#usage)
-3. [Support](#support)
-4. [Contributing](#contributing)
-5. [License](#license)
+1. [Support](#support)
+1. [Contributing](#contributing)
+1. [License](#license)
 
 ----
 
 ## Purpose
 
-##### What is Splunk Enterprise?
-Splunk Enterprise is a platform for operational intelligence. Our software lets you collect, analyze, and act upon the untapped value of big data that your technology infrastructure, security systems, and business applications generate. It gives you insights to drive operational performance and business results.
-
-Please refer to [Splunk products](https://www.splunk.com/en_us/software.html) for more knowledge about the features and capabilities of Splunk, and how you can bring it into your organization.
+#### What is Splunk Enterprise?
+Splunk Enterprise is a platform for operational intelligence. Splunk software lets you collect, analyze, and act upon the untapped value of big data that your technology infrastructure, security systems, and business applications generate. It gives you insights to drive operational performance and business results.
 
-##### What is splunk-ansible?
-This code in this repository is used for configuring Splunk Enterprise and Splunk Universal Forwarder instances based on a declarative configuration. The role of Ansible here enables managing Splunk in a manner consistent with industry standards such as infrastructure automation and infrastructure-as-code.
+See [Splunk products](https://www.splunk.com/en_us/software.html) for more information about the features and capabilities of Splunk products and how you can bring it into your organization.
 
-This repository should be used by people interested in configuring Splunk according to recommended best practices. The playbooks in this codebase are internally-vetted procedures and operations that administer and manage Splunk as done within the company.
+#### What is splunk-ansible?
+Use the code in this repository to configure Splunk Enterprise and Splunk Universal Forwarder instances based on a declarative configuration. You can use Ansible to manage Splunk Enterprise and Splunk Universal Forwarder in a manner consistent with industry standards such as infrastructure automation and infrastructure-as-code.
 
-## Usage
-For full usage instructions (including installation, tutorials, and examples), please visit the [splunk-ansible documentation](https://splunk.github.io/splunk-ansible/) page.
+The playbooks in this codebase are Splunk-vetted procedures and operations that administer and manage Splunk products as done within the company.
 
 ## Support
 Please use the [GitHub issue tracker](https://github.com/splunk/splunk-ansible/issues) to submit bugs or request features.
 
 If you have questions or need support, you can:
 * Post a question to [Splunk Answers](http://answers.splunk.com)
 * Join the [#docker](https://splunk-usergroups.slack.com/messages/C1RH09ERM/) room in the [Splunk Slack channel](http://splunk-usergroups.slack.com)
```

Comment:
Should we have internal references in a public repo?

Reply:
This is a separate slack channel for the public :)

Reply:
Didn't know we had public channels. TIL